src/App.jsx

```diff
@@ -1,39 +1,25 @@
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+  Navigate,
+} from 'react-router-dom';
 import { Provider } from 'react-redux';
 import { store } from './store/store.js';
-import { Routes, Route } from 'react-router'
-import {BrowserRouter as Router} from 'react-router-dom'
 
-import {HomePage} from './pages/HomePage'
-import {Header} from "./components/Header.jsx";
-import {Board} from "./components/board/Board.jsx";
-import { Sidebar } from 'components/sidebar/Sidebar';
+import { HomePage } from './pages/HomePage';
+import { Board } from './components/board/Board';
 
 export function App() {
   return (
     <Provider store={store}>
       <Router>
-          <div
-            className='surface'
-            style={{ backgroundColor: 'rgb(75, 191, 107)' }}
-          >
-            <Header />
-
-            <main>
-              <div className='container'>
-                <Sidebar />
-
-                <div className='content'>
-                  <Routes>
-                    {/* Public Routes */}
-                    <Route path='/' element={<HomePage />} />
-                    {/* Private Routes */}
-                    <Route path='/b/:id/:boardName' element={<Board />} />
-                  </Routes>
-                </div>
-              </div>
-            </main>
-          </div>
+        <Routes>
+          <Route path='/' element={<Navigate to='/home' replace />} />
+          <Route path='/home' element={<HomePage />} />
+          {/* <Route path='/board/*' element={<Board />} /> */}
+        </Routes>
```

Comment:
please fix the routing. add route to login and another route to board, maybe need to add a Layout.jsx component and there do a routing for the board.